osbs/utils/__init__.py

```diff
@@ -726,3 +726,41 @@ def copy(self):
             namespace=self.namespace,
             repo=self.repo,
             tag=self.tag)
+
+
+class UserWarningsStore(object):
+    def __init__(self):
+        # (asctime (platform:arch)? - name) - levelname - message
+        self.regex = r' - '.join((r'^.+', USER_WARNING_LEVEL_NAME, r'(\{.*\})$'))
```

Comment:
Why not `r'...'.format(USER_WARNIN.....)` ?

Reply:
Because I added additional brackets here, it requires an additional argument for this string. Here is the thrown exception: 
`>       self.regex = r'^.+ - {} -(\{.*\})$'.format(USER_WARNING_LEVEL_NAME)

E       IndexError: Replacement index 1 out of range for positional args tuple`

Reply:
`r'^.+ - {} -({{.*\}})$'.format(USER_WARNING_LEVEL_NAME)` This will work, but ok, it seems equally ugly